### ISSUE TITLE
Other geocoders?

### ISSUE THREAD

**Issue by tmcw** [tmcw-up-for-adoption/geocodify#3]
- Google's probably excluded because their geocoder TOS is very hard to follow.
- Yahoo's is probably good to include
- Maybe some local ones?
- Nominatim?

**Comment by scruss** [comment]
OpenMapQuest's nominatim server has much friendlier access than OSM, as it isn't so heavily constrained for CPU and bandwidth.

**Comment by tmcw** [comment]
Word, good call. I should also add twofishes, @blackmad is there a public instance of it?

**Comment by blackmad** [comment]
http://demo.twofishes.net

https://github.com/foursquare/twofishes/blob/master/docs/twofishes_requests.mddescribes
most of what you need to know. still need to document the
response formats. explore the space.

**Comment by tmcw** [comment]
MapQuest Open & Twofishes are in. Open specific tickets for other ones.
